packages/driver/cypress/fixtures/dangling-element.html

```diff
@@ -17,8 +17,7 @@
           return value
         },
         set (v) {
-          // eslint-disable-next-line no-console
-          console.warn(`tried to set frozen property ${property} with ${v}`)
+                    console.warn(`tried to set frozen property ${property} with ${v}`)
```

Comment:
This indentation seems quite large.

Reply:
Ya this was a search+replace screwup. Weirdly eslint doesn't handle indentation here, maybe cuz it's an HTML file? I reverted this in 966347aebd